mithril-aggregator/src/dependency_injection/builder.rs

```diff
@@ -1182,6 +1182,28 @@ impl DependenciesBuilder {
         Ok(self.signable_seed_builder.as_ref().cloned().unwrap())
     }
 
+    fn create_cardano_database_artifact_builder(
+        &self,
+        logger: &Logger,
+        cardano_node_version: Version,
+    ) -> CardanoDatabaseArtifactBuilder {
+        let local_uploader = LocalUploader::new(
+            self.configuration.get_server_url(),
+            &self.configuration.snapshot_directory.join("ancillary"),
```

Comment:
```suggestion
            &self.configuration.snapshot_directory.join("cardano-database").join("ancillary"),
```

Reply:
:warning: Nothing here ensure that this directory exist or create it otherwise.

Reply:
It has been handled here: https://github.com/input-output-hk/mithril/pull/2180/commits/1b10e78348ca18a3f196810dbbed666ac518d69b